config/routes.rb

```diff
@@ -614,20 +614,12 @@
 
     scope path: '/applications/:application_choice_id' do
       get '/' => 'application_choices#show', as: :application_choice
-      get '/offer' => 'application_choices#offer', as: :application_choice_offer
```

Comment:
Is it worth setting up a redirect so we don't kill existing links?

Reply:
I wonder if it's necessary as it's more likely for provider users to bookmark the candidate's application form if they want to come back rather than the offer tab, and there are no emails linking to offers.